src/core/src/bootstrap/Constants.py

```diff
@@ -258,12 +265,14 @@ class PatchAssessmentSummaryStartedBy(EnumBackport):
     class PatchOperationTopLevelErrorCode(EnumBackport):
         SUCCESS = 0
         ERROR = 1
+        WARNING = 2
 
     class PatchOperationErrorCodes(EnumBackport):
         DEFAULT_ERROR = "ERROR"  # default error code
         OPERATION_FAILED = "OPERATION_FAILED"
         PACKAGE_MANAGER_FAILURE = "PACKAGE_MANAGER_FAILURE"
         NEWER_OPERATION_SUPERSEDED = "NEWER_OPERATION_SUPERSEDED"
+        TRUNCATION = "PACKAGE_LIST_TRUNCATED"
```

Comment:
This is PatchOperationErrorCodes enum, since truncation is a warning type and not an error, this should not be included in here

Reply:
I think it's not because it's an error but because it sits inside the error object (even if it's not an "error" per se). Thoughts when viewed from that angle?